apps/webhooks.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: page
+key: apps-webhooks
+title: Webhooks
+---
+
+Webhooks allow developers to build apps that receive near-real-time information about events that happen in ePages shops.
+Instead of making constant API calls to check for specific events on the shop, you can register webhooks.
+So, directly after having registered for our [Developer Program](/#modal-popup), you might want to know more about how we have implemented webhooks.
+
+In order to receive the respective information about the events that your app subscribes to, webhooks require a callback URL.
+When the event happens, we'll send a `POST` request to your callback URL.
+Your app can then process the information send by us based on that event.
+
+Your app might require the following information:
+
+* an order has been created
+* an order has been updated
+* an order has been deleted
+* your app has been uninstalled.
+
+## Before you can get going
+
+Before you can send any requests or receive any responses, this is what you'll need:
+
+* Test shop: you can get a test shop by [joining the ePages Developer Program](/#modal-popup)
+* Client ID and Client Secret: received during [app creation](page:apps-create#get-your-credentials)
+* Access token: received during [app installation](page:apps-install)
+* SSL setup.
+
+## Ready to start creating a webhook
+
+To create a webhook, make a [`POST` request](page:apps-api-post-shopid-webhooks-information) to the respective webhook resource, including:
+
+* The app's `callbackUrl`
+* The `eventType`.
+
+By default, new webhooks will be set to **active**.
+If you want a webhook to be inactive initially, pass the following key-value `"active": false` with the request.
+
+An `HTTP 201` response indicates that the webhook was successfully set.
+
+## Webhook event types
+
+* APP_UNINSTALL
+* ORDERS_CREATE
+* ORDERS_UPDATE
+* ORDERS_DELETE
+
+## Receive webhook callbacks
+
+In order to receive the callback we send whenever events are triggered, you’ll need to create an application and configure your server.
+
+Depending on server traffic, notifications on events might take a few seconds.
+
```

Comment:
The structure of the notification is as follows:
```
{
   "id":"732EBF70-7138-426C-ABA4-E48AFDE3E47E",
   "baseUrl":"http://example.test.com/rs/shops/DemoShop",
   "entityUrl":"http://example.test.com/rs/shops/DemoShop/orders/58B6F171-3417-072A-0304-AC1532194AAB",
   "createdOn":"2017-03-20T10:58:18Z",
   "eventType":"ORDER_UPDATE",
   "entity":{...}
}
```
{...}
ist entweder das gleiche wie bei GET /products/id oder wie bei GET /orders/id